auth-manager/src/main/java/com/qpay/authmanager/model/entity/UserEntity.java

```diff
@@ -0,0 +1,41 @@
+package com.qpay.authmanager.model.entity;
+
+import com.qpay.libs.models.UserType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+import org.springframework.data.mongodb.core.mapping.FieldType;
+
+import java.time.LocalDateTime;
+
+@Document(collection = "users")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
remove @AllArgsConstructor and @NoArgsConstructor. But you will have to check manually, since you do not have integration tests, that nothing is broken